fabric-blockrenderlayer-v1/src/main/java/net/fabricmc/fabric/impl/blockrenderlayer/v1/BlockRenderLayerMapImpl.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.impl.blockrenderlayer.v1;
+
+import java.util.function.BiConsumer;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import net.fabricmc.fabric.api.blockrenderlayer.v1.BlockRenderLayerMap;
+import net.minecraft.block.Block;
+import net.minecraft.block.BlockRenderLayer;
+import net.minecraft.fluid.Fluid;
+
+public class BlockRenderLayerMapImpl implements BlockRenderLayerMap {
+	private BlockRenderLayerMapImpl() {}
+	
+	@Override
+	public void putBlock(Block block, BlockRenderLayer renderLayer) {
+		if (block == null) {
+			log.warn("Ignoring request to map null block to BlockRenderLayer");
```

Comment:
Not sure if crashing would be better here? Along with the dummy handlers.

Reply:
We'd be introducing a crash where one wouldn't normally occur.  

If the Mixin isn't triggered but doesn't crash on its own then there's been a big change by Mojang (or some other mod has broken it) and BlockRenderLayer may not even exist in the same way.  The worst thing that should happen then is some blocks/fluids wouldn't render correctly.  

Having it fail softly might give us a little more time to react to such changes but it probably doesn't matter and I'm open to changing it if you prefer.